src/Main/PlaneSliders.jsx

```diff
@@ -103,7 +133,7 @@ function PlaneSliders(props) {
   return (
     <div ref={planeSliders} className={slidersClass}>
       {planes.map((plane, idx) => {
-        return stateContextMain[`${plane}Slice`] ? (
+        return planeSlice2D[idx] ? (
```

Comment:
The slider disappears when the tick is dragged all the way to the left.  Use ` slicingPlanesVisibility[idx]` here instead of `planeSlice2D[idx]`.  

Can also use a `slicingPlanesVisibility[idx] && <div> ....</div>` style here instead of a ternary (?) operator.  Can then remove the empty `<div key={plane.toUpperCase() + 'hiddenSlider'} />`

Reply:
Ah, this is data dependent.  Try loading the `http://localhost:8082/test-data/HeadMRVolume.nrrd` file in the itk-vtk-viewer, move the slider tick all the way to the left.  The "slice" then becomes 0 and the slider goes away.

Reply:
Hey @PaulHax thank you so much for pointing this out 🙏🏼  
I just fixed this with a comparison using `!==null` instead, since the slicingPlanesVisibility[idx] starts off being false, which would mean that the whole thing would not initially render. Please let me know if you agree with this solution.

Reply:
Excellent!